regression/testdata/chaos-test-input.yml

```diff
@@ -0,0 +1,79 @@
+organizations:
+  - name: org1
+    connProfilePath: ./connection-profile
```

Comment:
Isn't this path supposed to target a specific yaml file?

Reply:
it supports a path or file. The example I based mine on targeted a path, so I kept with that